libpixman.mk

```diff
@@ -0,0 +1,55 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS       += libpixman
+LIBPIXMAN_VERSION := 0.40.0
+DEB_LIBPIXMAN_V   ?= $(LIBPIXMAN_VERSION)
+
+libpixman-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://cairographics.org/releases/pixman-$(LIBPIXMAN_VERSION).tar.gz
+	$(call EXTRACT_TAR,pixman-$(LIBPIXMAN_VERSION).tar.gz,pixman-$(LIBPIXMAN_VERSION),libpixman)
+
+ifneq ($(wildcard $(BUILD_WORK)/libpixman/.build_complete),)
+libpixman:
+	@echo "Using previously built libpixman."
+else
+libpixman: libpixman-setup libpng16
+	cd $(BUILD_WORK)/libpixman && ./configure -C \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=/usr \
+		--disable-dependency-tracking \
+		--disable-gtk \
+		--disable-silent-rules 
+	+$(MAKE) -C $(BUILD_WORK)/libpixman
+	+$(MAKE) -C $(BUILD_WORK)/libpixman install \
+		DESTDIR=$(BUILD_STAGE)/libpixman
+	touch $(BUILD_WORK)/libpixman/.build_complete
+endif
+
+libpixman-package: libpixman-stage
+	# libpixman.mk Package Structure
+	rm -rf $(BUILD_DIST)/libpixman{-dev}
+	mkdir -p $(BUILD_DIST)/libpixman/usr/lib \
+		$(BUILD_DIST)/libpixman-dev/usr/{lib,include/pixman-1}
+	
+	# libpixman.mk Prep libpixman
```

Comment:
Please keep debian naming scheme. Splitting isn't just about splitting, it's also about having the so_versions in the lib package name.

Reply:
So package them as `libpixman-1-0*`?

Reply:
Exactly what Debian names it.